invoices_app/views.py

```diff
@@ -147,6 +147,9 @@ def form_valid(self, form):
         payment_term = taxpayer.get_taxpayer_child().payment_term
         form.instance.invoice_due_date = form.cleaned_data['invoice_date'] + timedelta(days=payment_term)
         invoice_number = form.cleaned_data['invoice_number']
+        if taxpayer.taxpayer_state != 'ACTIVE':
```

Comment:
Este valor `ACTIVE` debería estar en una constante

Reply:
thanks, I added the constant.